docs/components/Form.md

```diff
@@ -169,6 +169,10 @@ Fortunately, VForm provides a `clear()` method.
       <button type="button" @click="form.clear">
         Clear
       </button>
+
+      <button type="submit" :disabled="!form.valid">
+        Submit
+      </button>
```

Comment:
Any time you have a `<form></form>` it must include either `<input type="submit">` or `<button type="submit"></button>` because people without keyboard access need a way to submit a form. So we either need to add them in, or remove the `<form></form>` wrapper. I have no preference, on most I added the button, on some I removed the form. Feel free to invert any of them.